plugins/ms-vscode.php.intelephense/1.0.13/meta.yaml

```diff
@@ -0,0 +1,14 @@
+id: ms-vscode.php.intelephense
+version: 1.0.13
+type: VS Code extension
+name: PHP
+title: PHP Intelephense
+description: PHP language features
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/bmewburn/vsextensions/vscode-intelephense-client/1.0.13/vspackage
+publisher: Red Hat, Inc.
+repository: https://github.com/bmewburn/vscode-intelephense
+category: Language
+firstPublicationDate: "2019-04-16"
+attributes:
+  containerImage: "eclipse/che-remote-plugin-php7:next"
```

Comment:
To add this plugin to more recent fields in the registry, could you add the following `meta.yaml` to paths `./v2/plugins/redhat/php/1.0.13` and `./v3/plugins/redhat/php/1.0.13`

```yaml
apiVersion: v2
publisher: redhat
name: php
version: 1.0.13
displayName: PHP Intelephense
title: PHP Intelephense
type: VS Code extension
icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
repository: https://github.com/bmewburn/vscode-intelephense
category: Language
firstPublicationDate: "2019-04-16"
extensions:
  - https://marketplace.visualstudio.com/_apis/public/gallery/publishers/bmewburn/vsextensions/vscode-intelephense-client/1.0.13/vspackage
```

The salient changes are: 
1. No `url` field; moved to `extensions`
2. `id` field deprecated; `id`s are now the combination `publisher/name/version` (updated name to `php`, publisher to `redhat` as that's what's used for other Red Hat published plugins.
3. Add `apiVersion` field as it's necessary for the v3 path.

Reply:
@amisevsk Should I change `apiVersion: v2`  to `apiVersion: v3` in your meta.yaml in case I'm going to target the `next release`?

Reply:
No, the `v2`, `v3` in this repo is unrelated to `v2`/`v3` in `meta.yaml` spec... I know it's confusing. We were talking today about a plan for improving docs around this.

For `meta.yaml`s, at the time of writing only `v2` is supported (in the `v3` folder of the repo...).